Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -260,7 +260,7 @@ extension TopSitesPanel: UICollectionViewDelegate {
         if let site = dataSource[indexPath.item] {
             // We're gonna call Top Sites bookmarks for now.
             let visitType = VisitType.Bookmark
-            let destination = NSURL(string: site.url)?.domainURL() ?? NSURL(string: "about:blank")!
+            let destination = NSURL(string: site.url) ?? NSURL(string: "about:blank")!
```

Comment:
This isn't right: it'll make *all* Top Sites load whichever full URL contributed the most to the tile.